Index: main-sf/timeline.py
===================================================================
--- main-sf.orig/timeline.py	2011-08-14 19:48:02.000000000 +0200
+++ main-sf/timeline.py	2011-08-14 19:49:04.000000000 +0200
@@ -18,30 +18,16 @@
 # along with Timeline.  If not, see <http://www.gnu.org/licenses/>.
 
 import gettext
-import locale
-import os
-import platform
 import sys
 
 # Make sure that we can import timelinelib
-sys.path.insert(0, os.path.dirname(__file__))
-# Make sure that we can import pysvg
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "libs", "dependencies", "pysvg-0.2.1"))
-# Make sure that we can import icalendar
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "libs", "dependencies", "icalendar-2.1"))
-# Make sure that we can import markdown
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "libs", "dependencies", "markdown-2.0.3"))
+sys.path.insert(0, "/usr/lib/timeline")
 
 from timelinelib.about import APPLICATION_NAME
 from timelinelib.arguments import ApplicationArguments
 from timelinelib.wxgui.setup import start_wx_application
 from timelinelib.paths import LOCALE_DIR
 
-if platform.system() == "Windows":
-    # The appropriate environment variables are set on other systems
-    language, encoding = locale.getdefaultlocale()
-    os.environ['LANG'] = language
-
 gettext.install(APPLICATION_NAME.lower(), LOCALE_DIR, unicode=True)
 
 application_arguments = ApplicationArguments()
