Index: timeline-0.4.0dev8d3caf448cdb/src/paths.py
===================================================================
--- timeline-0.4.0dev8d3caf448cdb.orig/src/paths.py	2009-08-16 16:50:08.000000000 +0200
+++ timeline-0.4.0dev8d3caf448cdb/src/paths.py	2009-08-16 16:50:53.000000000 +0200
@@ -24,7 +24,6 @@
 import os.path
 
 
-_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
-ICONS_DIR = os.path.join(_ROOT, "icons")
-LOCALE_DIR = os.path.join(_ROOT, "po")
-MANUAL_FILE = os.path.join(_ROOT, "manual", "manual.html")
+ICONS_DIR = "/usr/share/timeline/icons"
+LOCALE_DIR = "/usr/share/locale"
+MANUAL_FILE = "/usr/share/doc/timeline/manual/manual.html"
